### ISSUE TITLE
[feat]: slide through multiple images 

### ISSUE THREAD

**Issue by SimonGolms** [SimonGolms/ngx-ionic-image-viewer#21]
# Feature Request

**Ionic version:**

[x] **4.x**

**Describe the Feature Request**
When I tap on the image, it opens up the full-screen image one at a time. Let's say there is a gallery and I want to slide through multiple images when it is in the full-screen mode.

**Describe Preferred Solution**
The component recognizes the other `ion-img-viewer` elements on the side and shows the opportunity to slide to them.

**Describe Alternatives**
Provide an array to the property of `src` and slide to all of the images.

**Related Code**
```
<ion-img-viewer src="['image1', 'image2', 'image3]" activeIndex="0">
```

or

```
<ion-img-viewer src="image1">
<ion-img-viewer src="image2">
```

**Additional Context**
Alternative component: https://fivethree-team.github.io/ionic-4-components/gallery

**Comment by adeel399** [comment]
this feature is not working still

**Comment by adeel399** [comment]
i want the gallery img to open in viewer and then we can slide to next images but it does not work

**Comment by SimonGolms** [comment]
@adeel399 Indeed, that's basically what a feature request is all about. Since it's also labeled "help wanted", feel free to submit a relevant function enhancement.

**Comment by adeel399** [comment]
For now i gave made a custom model for image viewer but ..for future u should update it for multiple image and also option to add button in header like delete etc

**Comment by YaroslavG** [comment]
@SimonGolms ,
It still not working for multiple images ?

**Comment by annaformaniuk** [comment]
@SimonGolms Is there any update on this? Would indeed be a very much needed feature!
